### ISSUE TITLE
crancache_home

### ISSUE THREAD

**Issue by hadley** [r-lib/crancache#18]
Just prints the directory so you can inspect it locally if you want

**Comment by gaborcsardi** [comment]
There is get_cache_dir i think.

**Comment by hadley** [comment]
Yeah, it just needs to be exported with a name that matches the rest of the API

**Comment by hadley** [comment]
Maybe `crancache_status()`:

```R
crancache_status()
#> Directory: "/Users/hadley/Library/Caches/R-crancache"
#> File size: 500 meg
```

**Comment by gaborcsardi** [comment]
Yeah i was planning on adding an "info" or "status" method, but it updating
the info needs locking.
Either with `filelock`, or in a SQLite table.
